src/Caching/StackExchangeRedis/src/StackExchangeRedisCacheServiceCollectionExtensions.cs

```diff
@@ -32,8 +34,16 @@ public static IServiceCollection AddStackExchangeRedisCache(this IServiceCollect
         }
 
         services.AddOptions();
+
+#if NET7_0_OR_GREATER
+        services.AddLogging();
```

Comment:
I still don't think we should be adding logging at all in this extension method. I think just lighting up if it's there is fine. But out of curiosity, why the difference between .NET 7 and everything else?

Reply:
> But out of curiosity, why the difference between .NET 7 and everything else?

`AddLogging()` isn't available for `netstandard2.0` and `net472`.